code/game/machinery/computer/message.dm

```diff
@@ -30,10 +30,28 @@
 	var/obj/item/device/pda/customrecepient = null
 	var/customjob		= "Admin"
 	var/custommessage 	= "This is a test, please ignore."
+	var/switchboard_rows = 4
```

Comment:
this name is extremely confusing, because it's referring to columns not rows and also you start your counter at 0, meaning this is actually 5 columns

Reply:
i might have brain damage because i get the two wrong even in my native language